ydb/docs/en/core/development/load-actors-storage.md

```diff
@@ -79,4 +104,31 @@ StorageLoad: {
 
 When viewing test results, the following value should be of most interest to you:
 
-* ` ReadSpeed@ 100%`: 100 percentile of read speed in MB/s, e.g., `60.86`.
+* `ReadSpeed@ 100%`: 100 percentile of read speed in MB/s, e.g., `60.86`.
+
+### Read only {#readonly}
+
+Before the start of the main load cycle the `1 GB` data block of blobs with sizes between `1 MB` and `5 MB` is allocated. To avoid overloading the system with write requests the number of simultaneously processed requests is limited by the value of `5`. After completing the initial data aloocation the main cycle is launched, which consists of read requests sent with increasing rate: from `10` to `50` requests per second, the rate will increase gradually for `300` seconds.
+
+```proto
+StorageLoad: {
+    DurationSeconds: 300
+    Tablets: {
+        Tablets: { TabletId: 5000 Channel: 0 GroupId: 2181038080 Generation: 1 }
+
+        MaxInFlightReadRequests: 10
+        GetHandleClass: FastRead
+        ReadHardRateDispatcher {
+                RequestsPerSecondAtStart: 10
+                RequestsPerSecondOnFinish: 50
+        }
+
+        InitialAllocation {
+                TotalSize: 1000000000
+                BlobSizes: { Weight: 1.0 Min: 1000000 Max: 5000000 }
+                MaxWritesInFlight: 5
+        }
+    }
+}
+```
+Calculated percentiles will only represent the requests of the main load cycle and won't include write requests sent during the initial data allocation. The graphs in Monitoring should be of interest, for example, they allow to trace the request latency degradation caused by the increasing load.
```

Comment:
"Monitoring" probably needs to be a link to some instructions on where to look at